modules/dxp/apps/saml/saml-opensaml-integration/src/main/java/com/liferay/saml/opensaml/integration/internal/servlet/profile/WebSsoProfileImpl.java

```diff
@@ -2082,6 +2087,7 @@ private void _sendFailureResponse(
 
 		Response response = OpenSamlUtil.buildResponse();
 
+		response.setID(generateIdentifier(20));
```

Comment:
@jgarcialr sort this to keep `response.setID` after the `response.setDestination`. Thx